pinot-common/src/main/java/org/apache/pinot/common/function/scalar/ComparisonFunctions.java

```diff
@@ -61,6 +61,6 @@ public static boolean equals(double a, double b) {
 
   @ScalarFunction
   public static boolean between(double val, double a, double b) {
-    return val > a && val < b;
+    return val >= a && val <= b;
```

Comment:
Strange that we didn't have a test before. Could you add one?

Reply:
I think it's not used in normal filter workflow, but in intermediate stage.